src/debugger/channel/AdapterChannel.js

```diff
@@ -109,17 +109,25 @@ export class AdapterChannel {
   }
 
   launch(requestID: number, args: PrepackLaunchArguments, callback: DebuggerResponse => void) {
-    if (!args.prepackCommand || args.prepackCommand.length === 0) {
-      process.exit(1);
-    }
-
     this.sendDebuggerStart(requestID);
     this.listenOnFile(this._processPrepackMessage.bind(this));
 
-    let prepackArgs = args.prepackCommand.split(" ");
+    let prepackCommand = [args.sourceFile].concat(args.prepackArguments);
     // Note: here the input file for the adapter is the output file for Prepack, and vice versa.
-    prepackArgs = prepackArgs.concat(["--debugInFilePath", args.outFilePath, "--debugOutFilePath", args.inFilePath]);
-    this._prepackProcess = child_process.spawn("node", prepackArgs);
+    prepackCommand = prepackCommand.concat([
+      "--debugInFilePath",
+      args.outFilePath,
+      "--debugOutFilePath",
+      args.inFilePath,
+    ]);
+    // the runtime can be `prepack` or `node <script>`
```

Comment:
Can you explain why we are supporting two runtimes, instead of just prepack?

Reply:
Prepack can be started in different ways. e.g. the `prepack` command or `node lib/prepack-cli.js`.

Reply:
As we discussed offline, we just need user to specify the location of prepack and use the same consistent way to run it so we do not need separate runtime.